### DISCUSSION TITLE
installing EL1000 module

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#299] [Q&A] [answered]
I'm very sorry if I missed this -- I searched in [the docs](https://childproject.readthedocs.io/en/latest/search.html?q=el1000&check_keywords=yes&area=default), and in the [childproject code](https://github.com/LAAC-LSCP/ChildProject/search?q=el1000&type=code), as well as in the [EL1000 README](https://gin.g-node.org/EL1000/EL1000), and I didn't find how to install the EL1000 **module** (not the corpus):

> Traceback (most recent call last):
  File "scripts/cha_import.py", line 5, in <module>
    from EL1000.annotations import AnnotationImporter
ModuleNotFoundError: No module named 'EL1000'

`pip install EL1000`

>ERROR: Could not find a version that satisfies the requirement EL1000 (from versions: none)
ERROR: No matching distribution found for EL1000

**Comment by lucasgautheron** [comment, marked as answer]
/I/ am very sorry! I don't think I provided the instructions in the
appropriate place.

The following command should work:

pip install git+ssh://git@gin.g-node.org:/EL1000/tools.git --upgrade

**Comment by lucasgautheron** [comment]
EL1000 README updated: https://gin.g-node.org/EL1000/EL1000/#the-el1000-package
